the-collab-lab/tcl-75-smart-shopping-list
# - Initializating Firebase in project and setting up hosting

## Description

In this PR, additional files are being added to the project that were generated from initializing Firebase in the project and setting up hosting.

## Acceptance Criteria

- Followed instructions provided [here](https://thecollablab.notion.site/How-to-set-up-CI-CD-for-project-repos-3dc27b787d344218b74060d47532b0b6) to set up CI/CD for the project repo.

## Testing Steps / QA Criteria

- Create a PR and verify that the preview deploys are working.
- Merge the PR to ensure production deploys are working. By default, the production URL should be `<name of repo>.firebaseapp.com` (e.g., https://tcl-75-smart-shopping-list.firebaseapp.com/).
